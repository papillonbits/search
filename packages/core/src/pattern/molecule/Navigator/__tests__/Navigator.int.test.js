import { regular } from './Navigator.int.story'

describe('<Navigator />', () => {
  describe('Render', () => {
    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
