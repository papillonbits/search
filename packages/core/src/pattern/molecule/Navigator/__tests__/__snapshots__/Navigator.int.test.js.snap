// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigator /> Render must match regular 1`] = `
<div>
  <div
    class="navigator"
  >
    <div
      class="navigatorHeading Subhead"
    >
      <div
        class="Subhead-heading"
      />
    </div>
    <div
      class="tabnav"
    >
      <nav
        aria-label="List Cars"
        class="tabnav-tabs"
      >
        <a
          aria-current="page"
          class="tabnav-tab"
          href="/list"
        >
          List Cars
        </a>
      </nav>
    </div>
    <div
      class="navigatorHeading Subhead"
    >
      <div
        class="Subhead-heading"
      />
    </div>
  </div>
</div>
`;
